test/e2e_node/node_perf_test.go

```diff
@@ -111,6 +113,21 @@ var _ = SIGDescribe("Node Performance Testing [Serial] [Slow] [Flaky]", func() {
 		framework.Logf("Time to complete workload %s: %v", wl.Name(), perf)
 	}
 
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("ensure environment has enough CPU + Memory to run")
+		minimumRequiredCPU := resource.MustParse("15")
+		minimumRequiredMemory := resource.MustParse("48Gi")
```

Comment:
just a question: Where this requirements (15 CPU and 48GB RAM) come from?
I cannot find the reason from the PR message.

Reply:
@oomichi from the workload descriptions themselves
https://github.com/kubernetes/kubernetes/blob/f77de02aa30994dd082a60b018796970bf8de892/test/e2e_node/perf/workloads/npb_ep.go#L47-L55
https://github.com/kubernetes/kubernetes/blob/f77de02aa30994dd082a60b018796970bf8de892/test/e2e_node/perf/workloads/npb_is.go#L45-L54
https://github.com/kubernetes/kubernetes/blob/f77de02aa30994dd082a60b018796970bf8de892/test/e2e_node/perf/workloads/tf_wide_deep.go#L47-L55
The last one is only 16GiB, so please check all of them.

Reply:
Sorry for my late response, and thanks for your explanation.
I felt it would be better to put different memory condition for 16Gi but such balance (CPU core number vs. RAM size) could not be matter. At least, this addition makes better than current one.